cli/src/wallet.rs

```diff
@@ -42,11 +42,11 @@ pub async fn new_wallet(cli: WalletCli) -> Result<(Option<Wallet>, Option<String
                 migrate_stronghold_snapshot_v2_to_v3_command(path).await?;
                 return Ok((None, None));
             }
-            WalletCommand::New { alias } => {
-                let (wallet, account) = new_command(storage_path, snapshot_path, alias).await?;
+            WalletCommand::NewAccount { alias } => {
+                let (wallet, account) = new_account_command(storage_path, snapshot_path, alias).await?;
                 (Some(wallet), Some(account))
             }
-            WalletCommand::SetNode { url } => {
+            WalletCommand::SetNodeUrl { url } => {
                 let wallet = set_node_command(storage_path, snapshot_path, url).await?;
```

Comment:
I was assuming you'd change this one as well sorry, should have left another comment

Reply:
Sorry, I actually shoul've change it. Fixed